index.html

```diff
@@ -131,8 +140,7 @@ <h2>Sponsors</h2>
         </ul>
       </section>
 
-
-      <section>
+      <section class="footer">
```

Comment:
You could consider changing this section class to footer to make it more semantic instead of applying a footer class.